content/run.md

```diff
@@ -184,10 +184,17 @@ The compset chosen here is I2000Clm50BgcCropGs. Initialize the model for the yea
 Change data atmosphere domain (ATM_DOMAIN_PATH) and land domain (LND_DOMAIN_PATH) file paths following below commands in the case directory:
 
     [~/cases/scand]$./xmlchange ATM_DOMAIN_PATH=$MEREGDATA,LND_DOMAIN_PATH=$MYREGDATA
+    
+```{discussion} xml-files
+Manually editing the `*.xml`-files is **not** recommended!
+```
```

Comment:
Maybe say "not advised" instead of **not** recommended? As I think it is not _that_ dangerous?